raml-maven-plugin-it/pom.xml

```diff
@@ -3,16 +3,30 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
+    <parent>
+        <groupId>uk.gov.justice.maven</groupId>
+        <artifactId>raml-maven</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
+    </parent>
+
     <modelVersion>4.0.0</modelVersion>
-    <groupId>uk.gov.justice.maven</groupId>
     <artifactId>raml-maven-plugin-it</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
-
-    <!-- TODO add parent after extracting framewrok specific parent pom that does not contain raml-maven-plugin configuration-->
-    <!-- remember to manually update version -->
 
     <build>
         <plugins>
+            <!--
```

Comment:
Do we overwrite all the rules, or should we just remove the snapshot version rule?
- Not sure if there are multiple rules configured.